roseau/load_flow/models/buses.py

```diff
@@ -102,14 +109,14 @@ def voltages(self) -> np.ndarray:
             An array of the voltages.
         """
         # TODO use self.potentials with the check
-        if self.n == 3:
+        if "n" in self.phases:
+            return self._potentials[:-1] - self._potentials[-1]  # an, bn, cn
+        else:
             return np.asarray(
                 self._potentials[1] - self._potentials[0],  # ab
                 self._potentials[2] - self._potentials[1],  # bc
                 self._potentials[0] - self._potentials[2],  # ca
             )
```

Comment:
Ok with neutral but this method needs to be improved to handle buses with phase "ab", "bc" and "ca" for instance. In this case, we may have to transform the phase string into integers. Eg. `"bc"` ⇾ `(2,1)` ⇾ `self._potentials[2] - self._potentials[1]`